chart/sns-connector/README.md

```diff
@@ -0,0 +1,190 @@
+# OpenFaaS Pro SQS Connector
+
+The SNS connector can be used to invoke functions from an AWS SNS messages.
+
+See also: [Trigger functions from AWS SNS messages](https://docs.openfaas.com/openfaas-pro/sns-events/)
+
+## Prerequisites
+
+- Purchase a license
+
+  You will need an OpenFaaS License
+
+  Contact us to find out more [openfaas.com/pricing](https://www.openfaas.com/pricing)
+
+- Install OpenFaaS
+
+  You must have a working OpenFaaS installed.
+
+## Configure your secrets
+
+- Create the required secret with your OpenFaaS Pro license code:
+
+```bash
+$ kubectl create secret generic \
+    -n openfaas \
+    openfaas-license \
+    --from-file license=$HOME/.openfaas/LICENSE
+```
+
+- Create an AWS credentials secret:
+
+```bash
+$ kubectl create secret generic -n openfaas \
+  aws-sns-credentials --from-file aws-sns-credentials=$HOME/sns-credentials.txt
+```
+
+## Configure ingress
+
+To receive http calls from AWS SNS the callback url has to be publicly accessible.
+
+Install [cert-manager](https://cert-manager.io/docs/), which is used to manage TLS certificates.
+
+You can use Helm, or [arkade](https://github.com/alexellis/arkade):
+
+```bash
+arkade install cert-manager
+```
+
+Install ingress-nginx using arkade or Helm:
+
+```bash
+arkade install ingress-nginx
+```
+
+Create an ACME certificate issuer:
+
+```bash
+export EMAIL="mail@example.com"
+
+cat > issuer-prod.yaml <<EOF
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt-prod
+  namespace: openfaas
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: $EMAIL
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
+EOF
+```
+
+```bash
+kubectl apply -f issuer-prod.yaml
+```
+
+Create an ingress record for the sns-connector:
+
+```bash
+export DOMAIN="sns.example.com"
+
+cat > ingress.yaml <<EOF
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: sns-connector
+  namespace: openfaas
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/issuer: letsencrypt-prod
+  labels:
+    app: sns-connector
+spec:
+  tls:
+  - hosts:
+    - $DOMAIN
+    secretName: sns-connector-cert
+  rules:
+  - host: $DOMAIN
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: sns-connector
+            port:
+              number: 8081
+EOF
+```
+
+Apply the ingress resource:
+
+```bash
+kubectl apply -f ingress.yaml
+```
+
+## Configure values.yaml
```

Comment:
Repeat this in the docs, I like it and I think it would be useful for the customers to see it there under the list of bullets